### PR TITLE
zh-CN: update Glossary/ARIA

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/aria/index.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
 title: ARIA
 slug: Glossary/ARIA
+l10n:
+  sourceCommit: 66de1f47585ccf1539d5c1cd2b051de7bc19f59d
 ---
 
 {{GlossarySidebar}}
 
-**ARIA** (_Accessible Rich {{glossary("Internet")}} Applications_) 是向{{Glossary("HTML")}} 中添加语义和其他元数据的{{Glossary("W3C")}} 规范，以满足用户的辅助技术的需要。
+**ARIA** (_无障碍富{{glossary("Internet", "因特网")}}应用_) 是向{{Glossary("HTML")}} 中添加语义和其他元数据的{{Glossary("W3C")}} 规范，从而让 Web 内容与程序对残障人士更加友好。它又岂能帮助使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
```
Comment (yin1999) [inline files/zh-cn/glossary/aria/index.md:10]: 翻译成“富互联网应用”，保持上下文一致。此外，这段话还是和英文文档不一致的

```suggestion
**ARIA** (_无障碍富{{glossary("Internet", "互联网")}}应用_) 是向 {{Glossary("HTML")}} 中添加语义和其他元数据的{{Glossary("W3C")}} 规范，从而让 Web 内容与程序对残障人士更加友好。它又岂能帮助使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/glossary/aria/index.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
 
-例如，你可以将 `role="alert"`添加到{{HTMLElement("p")}} {{glossary("tag")}}以通知视力有问题的用户该信息是重要的（否则你可能通过文字颜色传达）。
+例如，你可以将 `role="alert"` 添加到 {{HTMLElement("p")}} {{glossary("tag", "标签")}}来告知视力有问题的用户，这段信息是重要的（要不然你应该是会通过文字颜色传达这个意思）。
 
 ## 参见
 
-- [ARIA](/zh-CN/docs/Web/Accessibility/ARIA)
+- [无障碍富互联网应用（ARIA）](/zh-CN/docs/Web/Accessibility/ARIA)
+- [Web 无障碍倡议（WAI）](https://www.w3.org/WAI/)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/aria/index.md:17]:
```suggestion
- W3C 上的 [Web 无障碍倡议（WAI）](https://www.w3.org/WAI/)
```

File: files/zh-cn/glossary/aria/index.md [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+- [WAI-ARIA 推荐](https://w3c.github.io/aria/)
+- [ARIA 创作实践指南 (APG)](https://www.w3.org/WAI/ARIA/apg/)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/aria/index.md:19]:
```suggestion
- [ARIA 创作实践指南（APG）](https://www.w3.org/WAI/ARIA/apg/)
```

File: files/zh-cn/glossary/aria/index.md (hunk 1 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/aria/index.md:10]: ARIA ([Accessible](https://developer.mozilla.org/en-US/docs/Glossary/Accessibility) Rich [Internet](https://developer.mozilla.org/en-US/docs/Glossary/Internet) Applications) is a [W3C](https://developer.mozilla.org/en-US/docs/Glossary/W3C) specification developed by [WAI](https://developer.mozilla.org/en-US/docs/Glossary/WAI) to make Web content and Web applications more accessible to people with disabilities. It especially helps with dynamic content and advanced user interface controls developed with HTML, JavaScript, and related technologies.

```suggestion
**ARIA** (_无障碍富{{glossary("Internet", "互联网")}}应用_) 是由 {{glossary("WAI")}} 开发的 {{Glossary("W3C")}} 规范，旨在向 {{Glossary("HTML")}} 中添加语义和其他元数据的，从而让 Web 内容与程序对残障人士更加友好。它尤其帮助使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
```

File: files/zh-cn/glossary/aria/index.md [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
 title: ARIA
 slug: Glossary/ARIA
+l10n:
+  sourceCommit: 66de1f47585ccf1539d5c1cd2b051de7bc19f59d
 ---
 
 {{GlossarySidebar}}
 
-**ARIA** (_Accessible Rich {{glossary("Internet")}} Applications_) 是向{{Glossary("HTML")}} 中添加语义和其他元数据的{{Glossary("W3C")}} 规范，以满足用户的辅助技术的需要。
+**ARIA** (_无障碍富{{glossary("Internet", "互联网")}}应用_) 是由 {{glossary("WAI")}} 开发的 {{Glossary("W3C")}} 规范，旨在向 {{Glossary("HTML")}} 中添加语义和其他元数据的，从而让 Web 内容与程序对残障人士更加友好。它尤其帮助使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
```
Comment (yin1999) [inline files/zh-cn/glossary/aria/index.md:10]:
```suggestion
**ARIA** (_{{Glossary("Accessibility", "无障碍")}}富{{glossary("Internet", "互联网")}}应用_) 是由 {{glossary("WAI")}} 开发的 {{Glossary("W3C")}} 规范，旨在让 Web 内容与程序对残障人士更加友好。它尤其有助于使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
```

File: files/zh-cn/glossary/aria/index.md [hunk 5]
```diff
@@ -1,14 +1,24 @@
...
 
-例如，你可以将 `role="alert"`添加到{{HTMLElement("p")}} {{glossary("tag")}}以通知视力有问题的用户该信息是重要的（否则你可能通过文字颜色传达）。
+例如，你可以将 `role="alert"` 添加到 {{HTMLElement("p")}} {{glossary("tag", "标签")}}来告知视力有问题的用户，这段信息是重要的（要不然你应该是会通过文字颜色传达这个意思）。
```
Comment (yin1999) [inline files/zh-cn/glossary/aria/index.md:12]:
```suggestion
例如，你可以将 `role="alert"` 添加到 {{HTMLElement("p")}} {{glossary("tag", "标签")}}来告知视力有问题的用户，这段信息是重要的（要不然你应该会通过文字颜色传达这个意思）。
```

File: files/zh-cn/glossary/aria/index.md [hunk 6]
```diff
@@ -1,14 +1,24 @@
...
 
 ## 参见
 
-- [ARIA](/zh-CN/docs/Web/Accessibility/ARIA)
+- [无障碍富互联网应用（ARIA）](/zh-CN/docs/Web/Accessibility/ARIA)
+- W3C 上的 [Web 无障碍倡议（WAI）](https://www.w3.org/WAI/)
+- [WAI-ARIA 推荐](https://w3c.github.io/aria/)
```
Comment (yin1999) [inline files/zh-cn/glossary/aria/index.md:18]:
```suggestion
- [无障碍富互联网应用（WAI-ARIA）](https://w3c.github.io/aria/)
```

File: files/zh-cn/glossary/aria/index.md [hunk 7]
```diff
@@ -1,14 +1,24 @@
...
+**ARIA** (_无障碍富{{glossary("Internet", "互联网")}}应用_) 是由 {{glossary("WAI")}} 开发的 {{Glossary("W3C")}} 规范，旨在向 {{Glossary("HTML")}} 中添加语义和其他元数据的，从而让 Web 内容与程序对残障人士更加友好。它尤其帮助使用 HTML、JavaScript 和相关技术开发的动态内容和高级用户界面控件。
 
-例如，你可以将 `role="alert"`添加到{{HTMLElement("p")}} {{glossary("tag")}}以通知视力有问题的用户该信息是重要的（否则你可能通过文字颜色传达）。
+例如，你可以将 `role="alert"` 添加到 {{HTMLElement("p")}} {{glossary("tag", "标签")}}来告知视力有问题的用户，这段信息是重要的（要不然你应该是会通过文字颜色传达这个意思）。
 
 ## 参见
 
-- [ARIA](/zh-CN/docs/Web/Accessibility/ARIA)
+- [无障碍富互联网应用（ARIA）](/zh-CN/docs/Web/Accessibility/ARIA)
+- W3C 上的 [Web 无障碍倡议（WAI）](https://www.w3.org/WAI/)
```
Comment (yin1999) [inline files/zh-cn/glossary/aria/index.md:17]:
```suggestion
- [W3C Web 无障碍倡议（WAI）](https://www.w3.org/WAI/)
```

**Review by yin1999 — COMMENTED** [review]
哥们，还是建议在改完内容以后和英文文档的内容对一下。“参见”部分改了以后还是有差异

**Review by yin1999 — APPROVED** [review]

**Pull request merged**